.github/pull_request_template.md

```diff
@@ -1,21 +1,24 @@
-- **Please check if the PR fulfills these requirements**
+### Please check if the PR fulfills these requirements
   - [ ] Tests for the changes have been added (for bug fixes/features)
```

Comment:
You changed the bullet points to H3 headers. Therefore, the body doesn't need to be indented anymore. Would you be so kind and remove the indentations?